torch/_inductor/runtime/triton_compat.py

```diff
@@ -69,7 +69,18 @@ def GPUTarget(
         def _log2(x: Any) -> Any:
             raise NotImplementedError
 
-    HAS_WARP_SPEC = hasattr(tl, "async_task")
+    def _triton_config_has(param_name: str) -> bool:
+        if not hasattr(triton, "Config"):
+            return False
+        if not hasattr(triton.Config, "__init__"):
+            return False
+        return param_name in inspect.signature(triton.Config.__init__).parameters
```

Comment:
I would rather we check against a triton version rather than inspecting signature tbh. But that's ok if we need something fast.